gazelle/python/target.go

```diff
@@ -99,9 +99,11 @@ func (t *targetBuilder) addResolvedDependency(dep string) *targetBuilder {
 	return t
 }
 
-// addVisibility adds a visibility to the target.
-func (t *targetBuilder) addVisibility(visibility string) *targetBuilder {
-	t.visibility.Add(visibility)
+// addVisibility adds visibility labels to the target.
+func (t *targetBuilder) addVisibility(visibility []string) *targetBuilder {
+	for _, item := range visibility {
+		t.visibility.Add(item)
+	}
```

Comment:
I thought that I'd be able to do

```go
t.visibility.Add(visibility...)
```

instead, but I would get this error:

```
python/target.go:104:19: cannot use visibility (variable of type []string) as type []interface{} in argument to t.visibility.Add
```

I'm relatively new to go and haven't use the `gods.TreeSet` package before, so I'm not sure why things don't work.

Reply:
You can follow the pattern from above. See `addModuleDependencies`.

Reply:
Hmm... looks like doing so would mean having to adjust the arg type from `[]string` to `*treeset.Set`, which would cascade to `generate.go`.

As-is, the diff is smaller and I'm always a fan of using built-in types when possible.

That said, LMK if you feel strongly that this func should be `*treeset.Set` and I'll be happy to change it.